guacamoleb.json

```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "Nora (eepymeowers)",
+			"oem": "OnePlus",
+			"device": "OnePlus 7",
+			"filename": "crDroidAndroid-15.0-20241213-guacamoleb-v11.0-BETA.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/guacamoleb/11.x/crDroidAndroid-15.0-20241213-guacamoleb-v11.0-BETA.zip/download",
+			"timestamp": 1734134185,
+			"md5": "894866bc482d65570cc226491916f226",
+			"sha256": "f8465834b4255e24ba248692f2d4ed9895381dd67fcc6756583879072fb200d6",
+			"size": 1196669088,
+			"version": "11.0",
+			"buildtype": "BETA",
+			"forum": "https://xdaforums.com/t/rom-official-15-guacamoleb-crdroid-11-beta-for-oneplus-7.4708051/",
```

Comment:
update to new forum post template
https://github.com/crdroidandroid/crdroid_features/blob/15.0/ForumPostTemplate